packages/demo-protobuf/types/demo.proto.d.ts

```diff
@@ -0,0 +1,2 @@
+declare const _default: '\nsyntax = "proto3";\npackage demo;\n\nmessage MsgDemo {\n  string example = 1;\n}\n';
```

Comment:
Wow, all the string made it into the type. We should widen the type to `string`

Reply:
Seems to be a bug in either TS or ESLint, which claims the type `string` is unnecessary because it can be inferred.

Reply:
But in ESLint I'd say. Ot an incompatibility between ESLint and TS since the widening rules of TS changed over time. `as string` works but

```ts
const proto: string = `
syntax = "proto3";
package demo;

message MsgDemo {
  string example = 1;
}
`;

export default proto;
```

should work as well.